app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -13,6 +13,13 @@ import Warning from './warning';
 import AnalyticsContext from '../context/analytics';
 import BarcodeAttentionWarning from './barcode-attention-warning';
 import FailedCaptureAttemptsContext from '../context/failed-capture-attempts';
+import { InPersonContext } from '../context';
+
+function formatWithStrongNoWrap(text: string): ReactNode {
+  return formatHTML(text, {
+    strong: ({ children }) => <strong style={{ whiteSpace: 'nowrap' }}>{children}</strong>,
```

Comment:
You may want to check if this conflicts with the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP). IIRC there may be a prohibition on inline styles.

Reply:
> You may want to check if this conflicts with the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP). IIRC there may be a prohibition on inline styles.

This might be okay from a CSP point-of-view, since React would programmatically assign the styles through `element.style.whiteSpace` (not assigning a `style="..."` string attribute), but there's a design system utility class we could use here as well.

```suggestion
    strong: ({ children }) => <strong className="text-no-wrap">{children}</strong>,
```

https://designsystem.digital.gov/utilities/text-styles/

Reply:
done in f80bf84e1e040da53ab65623daf590e680550c3c